openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -562,7 +562,7 @@ Upon receiving a successful Authorization Response, a Token Request is made as d
 The following are the extension parameters to the Token Request used in a Pre-Authorized Code Flow defined by this specification:
 
 * `pre-authorized_code`: The code representing the authorization to obtain Credentials of a certain type. This parameter MUST be present if the `grant_type` is `urn:ietf:params:oauth:grant-type:pre-authorized_code`.
-* `tx_code`: OPTIONAL. String value containing a Transaction Code. This value MUST be present if a `tx_code` object was present in the Credential Offer (including if the object was empty). The string value MUST consist of a maximum of eight numeric characters (the numbers 0 - 9). This parameter MUST only be used if the `grant_type` is `urn:ietf:params:oauth:grant-type:pre-authorized_code`.
```

Comment:
@paulbastian I thought we agreed to remove this restriction in the PR you did on tx_code..?

Reply:
yes, we agreed on this. When I did the PR, I searched for this sentence for 10 Minutes without success, so I thought that this had already gone in a previous PR :open_mouth:

Reply:
oh wow - happy we found it now!